.github/workflows/release.yml

```diff
@@ -0,0 +1,70 @@
+name: Release Events
+
+on:
+  release:
+    types: [published]
+
+permissions:
+  id-token: write
+  deployments: write
+
+jobs:
+  deploy-dev:
+    runs-on: ubuntu-latest
+    if: github.event.release.prerelease == true
+    environment:
+      name: dev
+    permissions:
+      id-token: write
+
+    steps:
+      - name: ⬇️ Set up code
+        uses: actions/checkout@v4
+        with:
+          show-progress: false
+
+      - name: 🚀 Deploy
+        uses: ./.github/actions/deploy
+        with:
+          identity_provider: ${{ secrets.IDENTITY_PROVIDER }}
+          service_account_email: ${{ secrets.SERVICE_ACCOUNT_EMAIL }}
+          project_id: ${{ secrets.PROJECT_ID }}
+          vpc: ${{ secrets.VPC }}
+          environment_variables: PORTER_DEVELOPMENT=1
+
+  deploy-prod:
+    runs-on: ubuntu-latest
+    if: github.event.release.prerelease == false
+    environment:
+      name: prod
+    permissions:
+      id-token: write
+
+    steps:
+      - name: ⬇️ Set up code
+        uses: actions/checkout@v4
+        with:
+          show-progress: false
+
+      - name: 🚀 Deploy
+        uses: ./.github/actions/deploy
+        with:
+          identity_provider: ${{ secrets.IDENTITY_PROVIDER }}
+          service_account_email: ${{ secrets.SERVICE_ACCOUNT_EMAIL }}
+          project_id: ${{ secrets.PROJECT_ID }}
+          vpc: ${{ secrets.VPC }}
+
+  notify:
+    name: Notifications
+    needs: deploy-prod
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+      issues: write
```

Comment:
**thought**: is there a way to run this only if one of the two above run? When the release fails this runs multiple times. Is that ok or does it add duplicate comments?

Reply:
Because of the "needs" prop, this should only run when `deploy-prod` is successful. I didn't think that we wanted it to run on deploys to staging for most projects. I'm missing the situation where the deploy fails and this would run multiple times.

Reply:
ok that looks great. I'm adding this to the turn project. I assumed with only a needs the job will be in a waiting state but paired with the if it will be skipped? I'm not sure if it's an improvement or not. What do you think?

```yaml
   if: github.event.release.prerelease == false
   needs: [deploy-prod]
```

Reply:
I don't believe that the additional if statement adds any value. If deploy-prod fails, then Notifications is marked as skipped. Here's an example: https://github.com/agrc/locate/actions/runs/11186751114

Reply:
thanks for demystifying that.